galley/pkg/config/source/kube/apiserver/status/controller.go

```diff
@@ -190,9 +195,11 @@ mainloop:
 		// If there are no other subfields left, also delete the status field
 		if st.desiredStatus != nil {
```

Comment:
this seems weird we are taking a `desiredStatus` and modifying it. Shouldn't we just change what we set the desired status to?